<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Автоматизация ИД</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="centralwidget_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="gb_left">
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QSplitter" name="gb_left_splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="vbl_templates">
            <item>
             <widget class="QLabel" name="label_structure_execdoc">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Структура проекта&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treewidget_structure_execdoc">
              <property name="contextMenuPolicy">
               <enum>Qt::DefaultContextMenu</enum>
              </property>
              <column>
               <property name="text">
                <string notr="true">Проект не загружен</string>
               </property>
              </column>
              <item>
               <property name="text">
                <string>Титульный лист</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Паспорт</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
               <item>
                <property name="text">
                 <string>ПТ-1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ПТ-2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ПТ-3</string>
                </property>
                <property name="checkState">
                 <enum>Checked</enum>
                </property>
               </item>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <layout class="QVBoxLayout" name="vbl_pages">
            <item>
             <widget class="QLabel" name="label_current_template">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Шаблон&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combox_templates"/>
            </item>
            <item>
             <widget class="QLabel" name="label_pages_template">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Страницы шаблона&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="lw_pages_template">
              <item>
               <property name="text">
                <string>Лист 1</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Лист 2</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="gb_center">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label_realview">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Результат в режиме реального времени&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPdfView" name="widget_pdf_view" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="gb_right">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label_variables">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Форма заполнения&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollarea_inputforms">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollarea_inputforms_layout">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>271</width>
             <height>657</height>
            </rect>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_recent_projects">
     <property name="title">
      <string>Недавние проекты</string>
     </property>
    </widget>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="menu_recent_projects"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_export_to_pdf"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_editors">
    <property name="title">
     <string>Редакторы</string>
    </property>
    <addaction name="action_edit_composition"/>
    <addaction name="action_edit_templates"/>
    <addaction name="action_edit_variables"/>
   </widget>
   <widget class="QMenu" name="menu_scale">
    <property name="title">
     <string>Масштаб</string>
    </property>
    <addaction name="action_zoomin"/>
    <addaction name="action_zoomout"/>
    <addaction name="action_zoomfitpage"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_editors"/>
   <addaction name="menu_scale"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="tb_main">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Панель инструментов</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_export_to_pdf"/>
   <addaction name="separator"/>
   <addaction name="action_edit_composition"/>
   <addaction name="action_edit_templates"/>
   <addaction name="action_edit_variables"/>
   <addaction name="separator"/>
   <addaction name="action_zoomin"/>
   <addaction name="action_zoomout"/>
   <addaction name="action_zoomfitpage"/>
  </widget>
  <widget class="QStatusBar" name="status_bar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_new">
   <property name="icon">
    <iconset theme="document-new" resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/add-file.svg</normaloff>:/white-icons/resources/white-icons/add-file.svg</iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/open.svg</normaloff>:/white-icons/resources/white-icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/save.svg</normaloff>:/white-icons/resources/white-icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/save.svg</normaloff>:/white-icons/resources/white-icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_zoomin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/zoom-in.svg</normaloff>:/white-icons/resources/white-icons/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_zoomout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/zoom-out.svg</normaloff>:/white-icons/resources/white-icons/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="toolTip">
    <string>Уменьшить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_edit_variables">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/text-editor.svg</normaloff>:/white-icons/resources/white-icons/text-editor.svg</iconset>
   </property>
   <property name="text">
    <string>Редактор переменных</string>
   </property>
   <property name="iconText">
    <string>Редактор переменных</string>
   </property>
   <property name="toolTip">
    <string>Редактор переменных</string>
   </property>
  </action>
  <action name="action_zoomfitpage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/zoom-fit-width.svg</normaloff>:/white-icons/resources/white-icons/zoom-fit-width.svg</iconset>
   </property>
   <property name="text">
    <string>По ширине</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_export_to_pdf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/export.svg</normaloff>:/white-icons/resources/white-icons/export.svg</iconset>
   </property>
   <property name="text">
    <string>Экспорт в PDF</string>
   </property>
  </action>
  <action name="action_edit_templates">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/template.svg</normaloff>:/white-icons/resources/white-icons/template.svg</iconset>
   </property>
   <property name="text">
    <string>Редактор шаблонов</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_edit_composition">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/white-icons/resources/white-icons/items-tree.svg</normaloff>:/white-icons/resources/white-icons/items-tree.svg</iconset>
   </property>
   <property name="text">
    <string>Редактор состава ИД</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPdfView</class>
   <extends>QWidget</extends>
   <header>PySide6/QtPdfWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
